### PR TITLE
Added travis intergration

### PR DESCRIPTION
**Pull request by sherm84**
I'm not sure if I did it right so feel free to fix it

### CODE REVIEW

File: .travis.yml [hunk 1]
```diff
@@ -0,0 +1,14 @@
+language: node_js
+node_js:
+  - "8.0"
+script:
+  - echo "Deploy!"
+install:
```
Comment (ronen-e) [inline .travis.yml]: You would probably need: 
```
    - npm install -g bower
    - npm install
    - bower install
```

File: .travis.yml [hunk 2]
```diff
@@ -0,0 +1,14 @@
...
+  - npm install -g firebase-tools
+  - npm install -g polymer-cli
+test:
+  - polymer test
+build:
+  - polymer build
+after_success:
+  - firebase deploy --token "$FIREBASE_TOKEN"
```
Comment (ronen-e) [inline .travis.yml]: Maybe we should use cache to speed up the build process
```
cache:
  directories:
    - "node_modules"
```

Comment (bennypowers) [reply .travis.yml]: Good idea.

**Comment by bennypowers** [comment]
LGTM

**Pull request merged**